### PR TITLE
remove-bower-dependency

### PR DESCRIPTION
**Pull request by jtrinklein**
### Review
 - [x] @boskya

### After Merge

run:

```
git checkout master
git pull
npm version patch
```

### note

If npm version is `2.13` or later npm version will likely fail to run postversion script, leaving you with pending commits in your repo.

If this is the case, run `npm run-script postversion` to finish the process.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -98,7 +102,7 @@ module.exports = function(grunt) {
             options: {
                 precision: 6,
                 sourceComments: false,
-                loadPath: ['./styles/', './bower_components/']
+                loadPath: ['./styles/', './bower_components/', './node_modules/']
```
Comment (boskya) [inline Gruntfile.js]: do we need ./bower_components any more?

**Comment by boskya** [comment]
Can we remove font-awesome and pygments from bower.json?

**Comment by jtrinklein** [comment]
with this change we can use npm or bower, i didn't want to remove the bower stuff because that's how ppm still uses it

**Comment by boskya** [comment]
ok good to go then.

**Pull request merged**